vidgear/gears/camgear.py

```diff
@@ -341,13 +341,6 @@ def __update(self):
             if self.__terminate.is_set():
                 break
 
-            if self.__threaded_queue_mode:
-                # check queue buffer for overflow
-                if self.__queue.full():
-                    # stop iterating if overflowing occurs
-                    self.__terminate.wait(timeout=0.000001)
-                    continue
-

```

Comment:
Haven't caused frames-buffer overflow due to `Queue.put()` blocking nature. It was an serious problem in deques. Nice.